packages/core/admin/admin/src/content-manager/components/RelationInput/RelationInput.js

```diff
@@ -74,22 +125,43 @@ const RelationInput = ({
               onMenuOpen={onSearchOpen}
               onMenuScrollToBottom={onSearchNextPage}
               placeholder={placeholder}
+              name={name}
             />
           </>
         }
         loadMore={
           !disabled &&
           labelLoadMore && (
-            <TextButton onClick={() => onRelationLoadMore()} startIcon={<Refresh />}>
+            <TextButton
+              disabled={relations.isLoading}
+              onClick={() => onRelationLoadMore()}
+              startIcon={
+                relations.isLoading ? (
```

Comment:
Did you confirm with Maeva if we want this behavior? This will lead to two spinners: one in the list and one here which I personally find a bit too much.

Edit: I see that you removed the other one instead, which would work for me :)

Reply:
yep we're sync with Maeva on this one :)

Reply:
The problem now is, that the button might not be visible anymore, if there isn't a next page ...

Reply:
I'm thinking about the case where the number of relations is smaller than the number of relations to display: in this case `hasNextPage()` will be false from the start and the element will never be visible.

Reply:
I see, but I'm still missing a piece I think:
if number of relations is smaller than the number of relations to display => will we have `hasNextPage()` before relations finished loading? then we don't have a way to show that relations are loading?
if that's the issue maybe we can find a conditional logic to display the loading button even when `hasNextPage()` is false but loading is true?

Reply:
Yeah, that could work I guess ... You'd need to make sure though that the loadMore doesn't appear clickable and maybe only show the loader?

Reply:
when the load more button is in loading state it's also disabled - so not clickable! I integrated the logic manually for now but I left a comment to replace the custom one with the one from the DS once we release it
does that sound good to you?